docs/hosting/configuration/configuration-examples/index.md

```diff
@@ -1,7 +1,9 @@
 ---
+title: Configuration examples index
+description: Index for configuration examples.
 contentType: overview
 ---
 
 # Configuration examples
 
-This section contains examples for how to configure n8n to solve particular use cases.
+This section contains examples for how to configure n8n to solve particular use cases.
```

Comment:
It might make sense to add a section TOC to this page to make it less empty. To do so, just add this code:

```
[[% import "_macros/section-toc.html" as sectionToc %]]

[[ sectionToc.sectionToc(page) ]]
```

You can see an example here: 
Example: https://docs.n8n.io/code/builtin/overview/
Source: https://raw.githubusercontent.com/n8n-io/n8n-docs/main/docs/code/builtin/overview.md